dse-next/download_cassandra.sh

```diff
@@ -6,7 +6,7 @@ set -euo pipefail
 # via the Cassandra Enhancement Proposal (CEP) process.
 #
 
-git clone -b vsearch https://github.com/datastax/cassandra.git cassandra
```

Comment:
We no longer have the vsearch branch

Reply:
not related to the rest of the PR though? just general cleanup?